src/dndbeyond/content-scripts/character.js

```diff
@@ -817,6 +826,14 @@ function rollAction(paneClass, force_to_hit_only = false, force_damages_only = f
                 damages.push("1d8");
                 damage_types.push("Blessed Strikes");
             }
+
+            // Charger Feat
+            // I don't love the action_name isn't "Lightning Launcher" here, but it removes code duplication of the containing if statement above, while keeping it Melee-only
+            if (character.getSetting("charger-feat") && !action_name.includes("Lightning Launcher")) {
```

Comment:
oh boy, yeah, that was confusing. Here's my suggestion, all the way up in line 744, I'd do : 
```js
const isMeleeAttack = action_name.includes("Polearm Master - Bonus Attack") || action_name.includes("Unarmed Strike") || action_name.includes("Tavern Brawler Strike")
            || action_name.includes("Psychic Blade") || action_name.includes("Bite") || action_name.includes("Claws") || action_name.includes("Tail")
            || action_name.includes("Ram") || action_name.includes("Horns") || action_name.includes("Hooves") || action_name.includes("Talons")
            || action_name.includes("Thunder Gauntlets");

if (isMeleeAttack || action_name.includes("Lightning Launcher")) {
....
```

And here, you can do `&& isMeleeAttack`, it would look nicer and easier to maintain long term than a negative check that excludes a previous if.

Reply:
I wish DnDBeyond just had them specified with the Melee or Ranged info still... But yeah, I'll do this